database/src/main/kotlin/no/nav/su/se/bakover/database/søknadsbehandling/SøknadsbehandlingPostgresRepo.kt

```diff
@@ -93,6 +94,7 @@ internal class SøknadsbehandlingPostgresRepo(
         val saksbehandler = stringOrNull("saksbehandler")?.let { NavIdentBruker.Saksbehandler(it) }
         val saksnummer = Saksnummer(long("saksnummer"))
         val fritekstTilBrev = stringOrNull("fritekstTilBrev") ?: ""
+        val behandlingsperiode = stringOrNull("behandlingsperiode")?.let { objectMapper.readValue<Behandlingsperiode>(it) }
```

Comment:
Jeg har aldri hørt fag si `behandlingsperiode`, så jeg stemmer for `stønadsperiode`.

Reply:
Det virker som behandlingsperiode aldri kan være null for noen av typene? Kan vi ikke da bare ha `string("behandlingsperiode")`

Reply:
Enig. I utgangspunktet skal perioden være satt for alle tilstander, med unntak av "start-tilstanden".. type, perioden settes etter at vi har opprettet behandlingen. Liker litt kommentaren din under - altså starte med å initialisere med søkandsdato + 12, det ville eliminert det ene tilfellet hvor den kan være null.